win/rl/pynethack.cc

```diff
@@ -274,6 +283,8 @@ PYBIND11_MODULE(_pynethack, m)
     /* NetHack constants. */
     mn.attr("ROWNO") = py::int_(ROWNO);
     mn.attr("COLNO") = py::int_(COLNO);
+    mn.attr("NLE_TERM_LI") = py::int_(NLE_TERM_LI);
```

Comment:
These should be the same as ROWNO at all times?

Reply:
No - you need a top line for text and 2 bottom lines for states etc.

Reply:
Ah right I confused it with LI and CO. But i feel we could use these names and be aligned with nethack and be more obscure to boot?